packages/code-du-travail-modeles/src/__test__/depart.mise.retraite.hotels.cafes.restaurants.spec.ts

```diff
@@ -0,0 +1,108 @@
+import Engine from "publicodes";
+import { mergeModels } from "../internal/merger";
+import {
+  DepartRetraiteReferences,
+  MiseRetraiteReferences,
+} from "./common/LegalReferences";
+import { getReferences } from "../utils/GetReferences";
+
+const engine = new Engine(mergeModels());
+describe("CC 1979", () => {
```

Comment:
On peut ajouter le nom de la régle pour savoir ce que l'on test sur la CC 1979
```suggestion
describe("Prévis de retraite pour la CC 1979", () => {
```